backend/src/api/users/users.controllers.js

```diff
@@ -0,0 +1,24 @@
+const UserDB = require('./user.model');
+const catchAsync = require('../../utils.js/catchAsync');
+
+const updateUsersController = catchAsync(async (req, res, next) => {
+  if (req.user && req.body) {
```

Comment:
I've meant to remove this `if` completely

Reply:
done)